src/common/reactor.cc

```diff
@@ -412,15 +412,18 @@ namespace Pistache::Aio
             }
 
             template <typename Func>
-            void forEachHandler(Func func) const
-            {
+            void forEachHandler(Func func)
```

Comment:
Same thing here. Up to you, but you could optionally keep the `const` qualifier.